go.mod

```diff
@@ -12,18 +12,137 @@ require (
 	github.com/google/go-github/v41 v41.0.0
 	github.com/migueleliasweb/go-github-mock v0.0.5
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.14.0
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.17.0
 	github.com/openshift/api v0.0.0-20200930075302-db52bc4ef99f
+	github.com/redhat-appstudio/service-provider-integration-scm-file-retriever v0.2.5
 	github.com/spf13/afero v1.6.0
 	github.com/stretchr/testify v1.7.0
-	github.com/tektoncd/pipeline v0.30.0
+	github.com/tektoncd/pipeline v0.32.1
 	github.com/tektoncd/triggers v0.17.1
-	golang.org/x/oauth2 v0.0.0-20211005180243-6b3c2da341f1
-	k8s.io/api v0.21.4
-	k8s.io/apimachinery v0.21.4
-	k8s.io/client-go v0.21.4
-	sigs.k8s.io/controller-runtime v0.9.5
+	golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
+	k8s.io/api v0.22.5
+	k8s.io/apimachinery v0.22.5
+	k8s.io/client-go v0.22.5
+	sigs.k8s.io/controller-runtime v0.10.3
 	sigs.k8s.io/yaml v1.3.0
 
 )
+
+require (
+	cloud.google.com/go v0.98.0 // indirect
```

Comment:
is this because all the other mods havent upgraded to 1.17?

Reply:
Dunno, maybe? I'll take a look.

Pulling in scm-file-retriever required us to bump a number of other dependencies (namely cause they use a newer client-go version), so it might be related to that?

Reply:
Updated.